src/tcgwars/logic/impl/gen4/DPTrainerKit.groovy

```diff
@@ -0,0 +1,151 @@
+package tcgwars.logic.impl.gen4;
+import tcgwars.logic.impl.gen4.DiamondPearl;
+import tcgwars.logic.impl.gen4.MysteriousTreasures;
+
+
+import static tcgwars.logic.card.HP.*;
+import static tcgwars.logic.card.Type.*;
+import static tcgwars.logic.card.CardType.*;
+import static tcgwars.logic.groovy.TcgBuilders.*;
+import static tcgwars.logic.groovy.TcgStatics.*
+import static tcgwars.logic.card.Resistance.ResistanceType.*
+
+
+import tcgwars.logic.card.*
+import tcgwars.logic.util.*;
+
+/**
+ * @author axpendix@hotmail.com
```

Comment:
Also I'm not sure if these author tags are formatted right. The groovy file I based this off of was an old one I implemented in 2019.

Reply:
The author javadoc tag doesn't really do anything anyway. I find the git history to have the same use and haven't bothered adding my email to any of the files I've worked on. So long as it isn't causing an error whatever you put there should be fine.

Reply:
Sounds good then.